client/app/widgets/Notifications/index.jsx

```diff
@@ -18,117 +18,122 @@ export type State = {
   signedInKey?: number,
 };
 
-export class Notifications extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { notifications: '', alreadyMounted: false, open: false };
-  }
+const Notifications = ({
+  element,
+}: Props) => {
+  const [notifications, setNotifications] = useState('');
+  const [alreadyMounted, setAlreadyMounted] = useState(false);
+  const [open, setOpen] = useState(false);
+  const [modalKey, setModalKey] = useState(undefined);
+  const [signedInKey, setSignedInKey] = useState(undefined);
 
-  componentDidMount() {
-    this.fetchNotifications();
-  }
+  const setBody = (notifications1: string[]) => {
+    // I have changed notifications to notifications1 in order to avoid no-shadow error.
+    let updatedNotifications = '';
+    notifications1.forEach((item: string) => {
+      updatedNotifications += `<div>${item}</div>`;
+    });
+    setNotifications(updatedNotifications);
+  };
 
-  changeTitle = (count: number) => {
+  const changeTitle = (count: number) => {
     let { title } = window.document;
     const eliminate = `${title.substr(0, title.indexOf(') '))})`;
     title = title.replace(eliminate, '');
     window.document.title = count === 0 ? title : `(${count}) ${title}`;
   };
 
-  getPusherKey = (signedInKey: number) => {
-    this.setState({ signedInKey });
-    const metaPusherKey = Array.from(
-      window.document.getElementsByTagName('meta'),
-    ).filter((item) => item.getAttribute('name') === 'pusher-key')[0];
-    const pusherKey = metaPusherKey.getAttribute('content');
-    const pusher = new window.Pusher(pusherKey, { encrypted: true });
-    const channel = pusher.subscribe(`private-${signedInKey}`);
-    channel.bind('new_notification', (response: any) => {
-      if (response && response.data) {
-        this.fetchNotifications();
-      }
-    });
-  };
-
-  setBody = (notifications: string[]) => {
-    let updatedNotifications = '';
-    notifications.forEach((item: string) => {
-      updatedNotifications += `<div>${item}</div>`;
-    });
-    this.setState({ notifications: updatedNotifications });
-  };
-
-  fetchNotifications = () => {
-    const { alreadyMounted, signedInKey } = this.state;
+  const fetchNotifications = () => {
+    setAlreadyMounted(alreadyMounted);
+    setSignedInKey(signedInKey);
     return axios
       .get('/notifications/signed_in')
       .then((response: any) => {
         if (response && response.data && response.data.signed_in !== -1) {
           if (response.data.signed_in !== signedInKey) {
-            this.getPusherKey(response.data.signed_in);
+            getPusherKey(response.data.signed_in);
```

Comment:
If you place `// eslint-disable-next-line no-use-before-define` above this line, you'll silence the linter error that's breaking on CircleCI. Then you shouldn't need the comment on line 75.

Reply:
@Manasa2850 I asked in [Slack](https://ifme.slack.com/archives/C725HPXFF/p1586841508019100) and Nishiki had a better solution. We can define change our function definitions to use the function keyword instead of the assigning them to variables. 

`const fetchNotifications = () => {` becomes ``

and `const getPusherKey = (signedInKey1: number) => {` becomes `function getPusherKey(signedInKey1: number)`

After I gave you non-working suggestions, I set up the repo so I could try things out. I ran the linter locally by running `eslint 'app/**/*.js' 'app/**/*.jsx' app/widgets/Notifications/index.jsx` so I'm pretty sure it'll work. (You can use that command to see what lint errors you have without actually pushing up a new commit).